lisa/tools/powershell.py

```diff
@@ -24,7 +24,9 @@ def run_cmdlet(
         sudo: bool = False,
         fail_on_error: bool = True,
     ) -> str:
-        result = self.run(cmdlet, force_run=force_run, sudo=sudo)
+        result = self.node.execute(
```

Comment:
Consider there may be `"` in the command already. So it needs to replace `"` with escape char.